README.md

```diff
@@ -4,34 +4,26 @@
 [![docs](https://docs.rs/redis-module/badge.svg)](https://docs.rs/redis-module)
 [![CircleCI](https://circleci.com/gh/RedisLabsModules/redismodule-rs/tree/master.svg?style=svg)](https://circleci.com/gh/RedisLabsModules/redismodule-rs/tree/master)
 
-# redismodule-rs
+# valkeymodule-rs
 
-This crate provides an idiomatic Rust API for the [Redis Modules API](https://redis.io/topics/modules-intro).
-It allows writing Redis modules in Rust, without needing to use raw pointers or unsafe code. See [here](https://docs.rs/redis-module/latest) for the most recent API documentation.
+This crate provides an idiomatic Rust API for the [Valkey Modules API](https://redis.io/topics/modules-intro).
+It allows writing Valkey modules in Rust, without needing to use raw pointers or unsafe code. See [here](https://docs.rs/redis-module/latest) for the most recent API documentation.
```

Comment:
```suggestion
It allows writing Valkey modules in Rust, without needing to use raw pointers or unsafe code. See [here](https://docs.rs/valkey-module/latest) for the most recent API documentation.
```
Assuming this will be where the documentation will be?

Reply:
thank you for catching.  Yes, it already exists.  I updated the URL